pkg/openvpn/assets/server-config.template

```diff
@@ -46,7 +46,7 @@ dev {{ .Device }}
 {{/* Add firewall rules to block all traffic originating from the shoot cluster.
      The scripts are run after the tun device has been created (up) or removed (down). */ -}}
 script-security 2
-up "/bin/vpn-server firewall --mode up --device {{ .Device }}{{ range .ShootNetworks }} --shoot-network={{ printf "%s" . }}{{ end }}"
+up "/bin/vpn-server firewall --mode up --device {{ .Device }}{{- $total := len .ShootNetworks }} --shoot-network={{- range $index, $element := .ShootNetworks }}{{ printf "%s" . }}{{ if ne $total (add1 $index) }},{{ end }}{{-  end }}"
```

Comment:
How about calculating the value in the `go` code instead of creating a loop in templating?

Reply:
If you prefer to calculate the array in the go code I can change that.

Reply:
From my perspective it should be easier to understand if it is done in `go` code. It took me some time to get this loop and the `add1` function.